commands/info/user-info.js

```diff
@@ -31,6 +32,8 @@ module.exports = class UserInfoCommand extends Command {
 		const member = args.member;
 		const user = member.user;
 
+		const userNames = await userName.findAll({where: {userid: user.id}});
```

Comment:
spaces before and after keys and values where needed.
example: `{ where: { key: value } }`